src/components/dialog/CreateIssueTemplateDialog.vue

```diff
@@ -0,0 +1,852 @@
+          <v-form v-model="formRelationTypeValid">
+            <v-alert type="info">Relation Type (Placeholder)</v-alert>
+          </v-form>
+        </template>
+        <template v-slot:item.7>
+          <v-form v-model="formVersionTemplatesValid">
+            <v-alert type="info">Component Version Templates (Placeholder)</v-alert>
+          </v-form>
+        </template>
+        <template v-slot:item.8>
+          <v-form v-model="formTemplateFieldsValid">
+            <v-alert type="info">Template Field Specifications (Placeholder)</v-alert>
+          </v-form>
+        </template>
+
+      </v-stepper>
+
+      <v-card-actions>
+        <DefaultButton variant="text" :disabled="step === 1" @click="previous">Previous</DefaultButton>
+        <v-spacer />
+        <DefaultButton variant="text" color="" @click="!isDirty && cancelCreateIssueTemplate()">
+                Cancel
+                <ConfirmationDialog
+                    v-if="isDirty"
+                    :title="`Discard Issue Template?`"
+                    :message="`Are you sure you want to discard this issue template?`"
+                    confirm-text="Discard"
+                    @confirm="cancelCreateIssueTemplate"
+                />
+        </DefaultButton>
+        <DefaultButton variant="text" color="primary" @click="next">{{ step === stepLabels.length ? 'Create' : 'Next' }}</DefaultButton>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useForm, defineField } from 'vee-validate';
+import * as yup from 'yup';
+import { fieldConfig } from '@/util/vuetifyFormConfig';
+import { onEvent } from '@/util/eventBus';
+import { computed } from "vue";
+import ConfirmationDialog from "./ConfirmationDialog.vue";
+import ExpandableCard from '../ExpandableCard.vue';
+
+
+import { iconList as baseIconList } from '../../assets/icons';
+
+const createIssueTemplateDialog = ref(false);
+const step = ref(1);
+const stepLabels = [
+  'General',
+  'Issue Types, Priorities & States',
+  'Assignment & Relation Types',
+  'Template Field Specifications'
+];
+
+
+
+
+const formGeneralValid = ref(false);
+const formIssueTypesValid = ref(true);
+const formIssuePrioritiesValid = ref(true);
+const formIssueStatesValid = ref(true);
+const formAssignmentTypeValid = ref(true);
+const formRelationTypeValid = ref(true);
+const formVersionTemplatesValid = ref(true);
+const formTemplateFieldsValid = ref(true);
+
+const expandedCardKey = ref<{
+  nameID: string
+  type: "type" | "priority" | "state"
+} | null>(null)
+
+const currentEditedName = ref<string>("");
+const currentEditedDescription = ref<string>("");
+const nameErrorMessage = ref<string>("");
+
+const currentEditedValue = ref<number>(0)
+const valueErrorMessage = ref<string>("")
+
+const currentEditedIsOpen = ref<boolean>(false)
+
+const currentEditedInverseName = ref<string>("")
+const inverseNameErrorMessage = ref<string>("")
+
+type IssueType = {
```

Comment:
I wonder if you could use the generated types from the input types from graphql here